src/views/HomePage.js

```diff
@@ -20,9 +22,23 @@ export const HomePage = () => {
   return (
     <HomePageStyle>
       <Wrapper column flex>
-        <Title text="Cohorts" />
-        <LineGraph />
-        {cohorts && <CohortList cohorts={cohorts} />}
+        {!isPending && !isFailure && (
+          <NotificationStyle align="center" alignSelf="center" blue fontLarge>
+            Loading...
+          </NotificationStyle>
+        )}
+        {isFailure && (
+          <NotificationStyle align="center" alignSelf="center" pink fontLarge>
+            Sorry, something went wrong. Please try again later.
+          </NotificationStyle>
+        )}
+        {isPending && !isFailure && (
+          <>
+            <Title text="Cohorts" />
+            <LineGraph />
+            {cohorts && <CohortList cohorts={cohorts} />}
+          </>
+        )}
```

Comment:
I think we should restructure this to an if-else structure this way only one condition will render at a time. example:
```
{
    isPending ? (
        <NotificationStyle align="center" alignSelf="center" blue fontLarge>
            Loading...
          </NotificationStyle>
    ) : isFailure ? (
        <NotificationStyle align="center" alignSelf="center" pink fontLarge>
            Sorry, something went wrong. Please try again later.
          </NotificationStyle>
    ) : (
        <>
            <Title text="Cohorts" />
            <LineGraph />
            {cohorts && <CohortList cohorts={cohorts} />}
          </>
    )
} 
```

Reply:
Same question as above. Why should we change it to if-else structure? What's the advantage of it?

Reply:
**The advantages are:**
- handling the conditions in order as they are built **on top of each other** and not as separated sections. Then this will allow presenting **only** the things that need to be rendered and you will not need to state the conditions multiple times in the code. 
- using && is for simple conditions and as the code becomes longer it's harder to debug (which in this case if you will look closer the condition of **presenting data** is incorrect as you expect `isPending` to be true)

I tried to explain this better in the following diagram: 
![Screen Shot 2019-12-06 at 14 36 26](https://user-images.githubusercontent.com/52331753/70351010-f7ec1900-1835-11ea-9dd0-0d6b0c5051ca.png)